resources/web/wwi/equirectangular_to_cube_generator.js

```diff
@@ -0,0 +1,109 @@
+/* global THREE */
+
+// Source: https://github.com/mrdoob/three.js/blob/r104/examples/js/loaders/EquirectangularToCubeGenerator.js
+// @author Richard M. / https://github.com/richardmonette
+// @author WestLangley / http://github.com/WestLangley
+
+THREE.EquirectangularToCubeGenerator = (function() {
+  var camera = new THREE.PerspectiveCamera(90, 1, 0.1, 10);
+  var scene = new THREE.Scene();
+  var boxMesh = new THREE.Mesh(new THREE.BoxBufferGeometry(1, 1, 1), getShader());
+  boxMesh.material.side = THREE.BackSide;
+  scene.add(boxMesh);
+
+  class EquirectangularToCubeGenerator {
+    constructor(sourceTexture, options) {
+      options = options || {};
+
+      this.sourceTexture = sourceTexture;
+      this.resolution = options.resolution || 512;
```

Comment:
> I didn't find yet where the HDR equirectangular background texture is limited.

Not sure, but isn't it here?

Reply:
No, this line simply set a lower resolution, i.e. if `options.resolution > 512` then `this.resolution = options.resolution`